kubernetes/cray-opa/templates/policies/keycloak-admin.yaml

```diff
@@ -128,12 +128,24 @@ data:
           {"method": "PATCH",  "path": `.*`},
           {"method": "HEAD",  "path": `.*`},
       ],
+      "tenant-admin": [
```

Comment:
I called out the following endpoints in the design document, but I don't seem them in this role.
/v2
/v2/version
/v2/healthz
I feel like those are useful for ascertaining BOS versions and healthiness.

Reply:
Yep; I can add those!